sql/report_queries/costs/invoices.sql

```diff
@@ -0,0 +1,151 @@
+/**
+
+PURPOSE
+
+This query provides a summary of what was spent by purchase order line  
+on an invoice line across all the invoices within a particular time period. 
+Invoice line adjustments are included in the invoice line total. 
+This query only shows invoice line data that is linked to a purchase order.
```

Comment:
Could this be moved to a `README.md` file?

Reply:
Yes, I think so. It was thought as a working draft too and not loose any thoughts on it.

Reply:
I will create readme for the invoices query and move the documentation from the query into that file.